### PR TITLE
Apply Tint to Particle Container (#3899)

### PR DESCRIPTION
**Pull request by YahorPadlucki**
Applies tint to whole particle container.

### CODE REVIEW

File: src/particles/webgl/ParticleRenderer.js [hunk 1]
```diff
@@ -153,6 +154,7 @@ export default class ParticleRenderer extends core.ObjectRenderer
 
         this.shader.uniforms.projectionMatrix = m.toArray(true);
         this.shader.uniforms.uAlpha = container.worldAlpha;
+        this.shader.uniforms.tint = hex2rgb(container.tint);
```
Comment (bigtimebuddy) [inline src/particles/webgl/ParticleRenderer.js]: `hex2rgb` creates a new Array every `render` call here, which isn't good performance-wise. Maybe consider adding a private `_tintRGB` property on the ParticleContainer when `tint` is set? 

```js
set tint(value)
{
     this._tint = value;
     hex2rgb(value, this._tintRGB);
}
```
And where this is added to the constructor before the initial `this.tint = 0xFFFFFF`.
```js
this._tintRGB = []; 
```

Then this line could be `this.shader.uniforms.int = container._tintRGB`

Comment (YahorPadlucki) [inline src/particles/webgl/ParticleRenderer.js]: Thanks for the explanation - I've added _tintRGB property

**Comment by themoonrat** [comment]
A comment....
All current 'tint' properties are on classes that contain and can apply to a texture, and thus works on both canvas and webgl.
This same 'tint' property occurs on a container, and thus on is webgl only.

So I think the documentation for the property should be clear on this.

**Comment by bigtimebuddy** [comment]
Thanks for the doc changes @YahorPadlucki. Look okay to you @themoonrat?

**Pull request merged**